lab01/baseball.sql

```diff
@@ -67,4 +67,19 @@ inner join above_avg_sho as b on a.playerID = b.playerID;
 -- drop view maxSalary;                       
 -- drop view player_with_at_leaset_1_homerun;
 -- drop view r;                  
--- drop view sum_of_player;
\ No newline at end of file
+-- drop view sum_of_player;
+
+-- Questions 2)
+-- please change the file path when testing
+LOAD DATA LOCAL INFILE  '/Users/brickbai/Downloads/Fielding.csv'
```

Comment:
Include the Fielding.CSV file in the commit and can you test it with the relative path?

Reply:
According to [a post on Piazza](https://piazza.com/class/jqnxz0eifzr5g1?cid=20), "Do not worry about the path of the Fielding.csv file, graders will adjust it before running your script."